### PR TITLE
Expose more props to docs

### PR DESCRIPTION
**Pull request by benoitgrelard**

### CODE REVIEW

File: packages/radix-ui-themes/src/components/progress.tsx [hunk 1]
```diff
@@ -13,9 +13,7 @@ type ProgressOwnProps = GetPropDefTypes<typeof progressPropDefs>;
 interface ProgressProps
   extends Omit<PropsWithoutRefOrColor<typeof ProgressPrimitive.Root>, 'children'>,
     MarginProps,
-    ProgressOwnProps {
-  duration?: string;
-}
```
Comment (vladmoroz) [inline packages/radix-ui-themes/src/components/progress.tsx:18]: Should we do specify the duration type so that `${number}s` is enforced?

**Review by vladmoroz — APPROVED** [review]

File: packages/radix-ui-themes/src/components/progress.tsx (hunk 1 above)
Comment (vladmoroz) [reply packages/radix-ui-themes/src/components/progress.tsx:18]: We can keep it as 'string' in the docs

Comment (benoitgrelard) [reply packages/radix-ui-themes/src/components/progress.tsx:18]: you mean, override the type in the component?

Comment (vladmoroz) [reply packages/radix-ui-themes/src/components/progress.tsx:18]: Yeah potentially

**Pull request merged**